eng/common/pipelines/templates/steps/daily-dev-build-variable.yml

```diff
@@ -22,3 +22,12 @@ steps:
     echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
   displayName: "Setup Versioning Properties"
   condition: eq(variables['SetDevVersion'], '')
+- pwsh: |
+    $setDefaultBranch = "master"
+    if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
```

Comment:
I assume this is just for testing but yo uwill want to remove this condition and we should move this entire step into its own template.

Reply:
I am trying to move this step to daily-dev-build-variable.yml. However, it required to have the script everytime we used link-checker.yml. If we added the template before link-checker, then we have to set extra variables before using the link checker.